src/platform/intel/cavs/lib/clk.c

```diff
@@ -293,18 +293,29 @@ static int clock_platform_set_cpu_freq(int clock, int freq_idx)
 
 void platform_clock_init(struct sof *sof)
 {
+	uint32_t platform_waiti_clock = 1;
```

Comment:
Probably better to set this to CPU_WAITI_FREQ_IDX here otherwise it can be mistaken for magic number.

Reply:
yeap, much better, fixed